content/achievements.md

```diff
@@ -19,6 +19,16 @@ These ones were quite successful:
     <h2 class="timelineMajorMarker"><span>2018</span></h2>
     <dl class="timelineMinor">
 
+      <dt id="event34"><a>RoboCup Junior European Championship</a></dt>
```

Comment:
Please add the location `RoboCup Junior European Championship Pescara`

Reply:
OK, I'll do it

Reply:
Montesilvano is the town in Pescara province